app/src/main/res/values/tracker.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="ga_trackingId">UA-42015512-1</string>
```

Comment:
We had this implementation before. I don't know what is the best. I think if we use google's plugin, we should also use it for analytics. What do you think?